src/ServicePulse.Host/vue/src/assets/main.css

```diff
@@ -2395,7 +2395,7 @@ footer span.connection-failed.ng-scope {
 
 
 .pa-connection-establishing {
-    background-image: url('../img/loader_spinner.gif');
+    background-image: url('../a/img/loader_spinner.gif');
```

Comment:
We should copy this image to the Vue assets folder

Reply:
There are a many such image files in the vue main css. I was planning to have all of them copied over late on. For now, I have copied the images of the ones mentioned in this PR .